src/main/resources/db/migration/V17__added_update_agreement_status_types.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TYPE agreement_status_type
```

Comment:
does this need some form of "if not exist" ?

Reply:
Updated in [0936bed](https://github.com/ministryofjustice/hmpps-sentence-plan/pull/196/commits/0936bed536d0a76df2ab25ac1a0fdd517779bd64)